README.md

```diff
@@ -139,16 +139,47 @@ This means you shouldn't change any css files, but `.styl` files. They're in /st
 
 Autocompiling of `.styl` files to `.css`:
 
-    gulp watch
+    npx gulp watch
```

Comment:
Best practice is not to install CLI tools globally; `npx` will use the local binary automatically (instead of having to type `./node_modules/.bin/gulp watch`).